.github/workflows/check-issues-nightly.yml

```
@@ -0,0 +1,23 @@
+name: Check issues with pending validation labels
+
+on:
+  schedule:
+    - cron: '0 0 * * *' # 0:00 AM UTC time every day
```

```suggestion
    - cron: '15 1 * * *' # 1:15 AM UTC time every day
```

0:00 would be busy time, i think we could try to adjust the time a little bit.

---

That makes sense, I've updated it to 8:00 AM UTC (01:00 AM PST and 04:00 EST)